fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -40,15 +72,167 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
     ): View = ComposeView(requireContext()).apply {
         setContent {
             FirefoxTheme {
-                TranslationsDialogBottomSheet(
-                    onSettingClicked = {
-                        findNavController().popBackStack()
-                        findNavController().navigate(
-                            TranslationsDialogFragmentDirections.actionGlobalToTranslationOptionsDialogFragment(),
+                var translationsVisibility by remember {
+                    mutableStateOf(args.translationsDialogAccessPoint == TranslationsDialogAccessPoint.Translations)
+                }
+
+                val density = LocalDensity.current
```

Comment:
Perhaps reorder this so we can group all the variables with `remember {}` first and then our `density` variable

Reply:
Done